node/tests/st/bgp/test_ipip.py

```diff
@@ -283,8 +293,8 @@ def assert_tunl_ip(self, host, ip_network, expect=True):
                        interface.
         :return:
         """
-        retries = 7
```

Comment:
Hm, is the bump from 7 -> 8 meaningful? Do we need more tolerance in this retry?

Reply:
The docstring mentioned 7 retries, but in the code we were doing `range(retries + 1)`, which starting from 0 means 1 try + 8 retries, I was just trying to make this cleaner without changing the current behavior, so I bumped the number to 8 and removed the `+ 1` (while also changing the docstring). I wasn't sure if I could just do away with the `+ 1` and leave it at 7, but if you think that's better, I can just do that as well.